StoreTestWPF.ViewModel/ViewModels/StoreViewModel.cs

```diff
@@ -0,0 +1,160 @@
+﻿using StoreTestWPF.DAL;
+using StoreTestWPF.DAL.Models;
+using StoreTestWPF.ViewModel.Enums;
+using StoreTestWPF.ViewModel.Interfaces;
+using StoreTestWPF.ViewModel.Utils;
+using System;
+using System.Collections.ObjectModel;
+using System.Data.Entity;
+using System.Linq;
+using System.Windows.Input;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public sealed class StoreViewModel : ViewModelBase
+    {
+        private CakeViewModel selectedCake;
+        private bool isCakeSelected;
+        private ICommand loadCommand;
+        private ICommand addCommand;
+        private ICommand editCommand;
+        private ICommand deleteCommand;
+        private IViewService viewService;
```

Comment:
Should be readonly as well as dbContext.

Reply:
Fixed